lfucg-exactions/client/js/components/SubdivisionSummary.js

```diff
@@ -69,6 +71,50 @@ class SubdivisionSummary extends React.Component {
                 );
             })(plats);
 
+        const platIds = plats && plats.length > 0 && map(plat => plat.id)(plats);
```

Comment:
Are you trying to set platIds to `true` or `false`?
Looking at ramda map, I'm assuming you're passing in an array of plat objects and are trying to return an array of just the plat ids, but wouldn't the `&&` just be checking if that's `true` or `false` and if so, platIds would only be `true` or `false`.

Reply:
Essentially the && is waiting for the previous stuff to comeback as true and will set it equal to the last entry-- at least that is the way I have been using it. I got it from @KellyInnovation so maybe she would have a better technical description. PlatIDs in this case is being set to an array of numbers that correspond to plats. Make sense?

Reply:
I believe this is a valid way of doing this in this context.  Some other languages would behave as Elton described though, so be careful using this.